src/app/objects/resume.ts

```diff
@@ -0,0 +1,57 @@
+import { template } from "./template";
+
+export interface resume {
+  template: template,
+  header: header,
+  workExps: workExp[],
+  educations: education[],
+  certificates?: string[],
+  skills: string[],
+  summary: string
+}
+
+export interface header {
+  id: number;
+  firstName: string;
```

Comment:
I think it can be just a name instead of first and last names. Kai mentioned that he wanted to be able to accommodate different orders for different cultures. I think just using name might suit this case better

Reply:
Fixed